app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -175,9 +182,20 @@ class EventDetailsFragment : Fragment() {
         startActivity(intent)
     }
 
-    private fun reportEvent(event: Event){
-        val email ="support@eventyay.com"
-        val subject ="Report of ${event.name} (${event.identifier})"
+    private fun contactHost(event: Event) {
+        val email = event.paypalEmail
```

Comment:
@iamareebjamal right now I am using the paypal email because there is no email of the organizer. Should I create an issue in the server to store the email of the organizer?

Reply:
Yes, raise the issue to server guys

Reply:
@nikit19 Please see my comment on https://github.com/fossasia/open-event-server/issues/4952. If that works for you then fine otherwise I will update the server.

Reply:
@srv-twry yeah as soon as this https://github.com/fossasia/open-event-server/issues/4928 issue gets solved. This can be done

Reply:
@nikit19 That issue has been solved. You can proceed.